scripts/files/file_tiff.py

```diff
@@ -44,7 +44,9 @@ def set_scale(self, scale: int) -> None:
     def set_path_standardised(self, path: str) -> None:
         self._path_standardised = path
 
-    def get_gdalinfo(self) -> Optional[GdalInfo]:
+    def get_gdalinfo(self, original: Optional[bool] = False) -> Optional[GdalInfo]:
```

Comment:
In that case I won't bother handling `original` in that function but make `get_gdalinfo_original` accessible and call it instead.